src/input_common/drivers/udp_client.cpp:116:18: error: no type named 'io_service' in namespace 'boost::asio'
  116 |     boost::asio::io_service io_service;
      |     ~~~~~~~~~~~~~^
src/tests/input_common/calibration_configuration_job.cpp:85:18: error: no type named 'io_service' in namespace 'boost::asio'
   85 |     boost::asio::io_service io_service;
      |     ~~~~~~~~~~~~~^

--- src/input_common/drivers/udp_client.cpp.orig	2024-03-01 07:57:00 UTC
+++ src/input_common/drivers/udp_client.cpp
@@ -113,7 +113,7 @@ class Socket { (private)
     }
 
     SocketCallback callback;
-    boost::asio::io_service io_service;
+    boost::asio::io_context io_service;
     boost::asio::basic_waitable_timer<clock> timer;
     udp::socket socket;
 
--- src/tests/input_common/calibration_configuration_job.cpp.orig	2024-03-01 07:57:00 UTC
+++ src/tests/input_common/calibration_configuration_job.cpp
@@ -82,7 +82,7 @@ class FakeCemuhookServer { (private)
     }
 
 private:
-    boost::asio::io_service io_service;
+    boost::asio::io_context io_service;
     boost::asio::ip::udp::socket socket;
     std::array<u8, InputCommon::CemuhookUDP::MAX_PACKET_SIZE> send_buffer;
     std::array<u8, InputCommon::CemuhookUDP::MAX_PACKET_SIZE> receive_buffer;
